app/views/map/index.html.erb

```diff
@@ -1 +1,243 @@
-<div class="map"></div>
+<!DOCTYPE html>
+<html>
+  <head>
+  	<title>HPI Map</title>
+	<meta name="viewport" content="initial-scale=1,maximum-scale=1,user-scalable=no" />
+  	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.3.1/leaflet.css" />
+    <script src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js" integrity="sha512-XQoYMqMTK8LvdxXYG3nZ448hOEQiglfqkJs1NOQV44cWnUrBc8PkAOcXy20w0vlaXaVUearIOBhiXZ5V3ynxwA==" crossorigin=""></script>
+	<style>
+		/* .marker_label {
+			position: absolute;
+			background-color: transparent;
+			border: None;
+			box-shadow: None;
+			color: white;
+			font-size: 100px;
+		} */
+		.leaflet-tooltip.marker_label{
+			border-radius: 20px;
+			background-color: transparent;
+			padding-left: 0px;
+			padding-right: 0px;
+			font-weight: bold;
+			border: None;
+			color: white;
+			box-shadow: None;
+			font-family: Roboto, "Open Sans", sans-serif;
+		}
+		.leaflet-tooltip-top:before, 
+		.leaflet-tooltip-bottom:before, 
+		.leaflet-tooltip-left:before, 
+		.leaflet-tooltip-right:before {
+			border: none !important;
+		}
+	</style>
+  </head>
+
+  <body>
+  <div class="leaflet">
+  	<div id="map" style="width: 100%; height: 600px;"></div>
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/leaflet.locatecontrol@0.76.0/dist/L.Control.Locate.min.css" />
```

Comment:
Put all imports (both css with link and js with script tags) in the same place, so at the top of this file

Reply:
put all includes in header and moved script to a `map_build.js` in assets, moved css to `map.scss` in assets, see latest commit

Reply:
except the `<%= javascript_include_tag 'OutdoorMap/map_build' %>` because this has to be after the declaration of `<div id="map" style="width: 100%; height: 600px;"></div>`